airbyte-connector-builder-server/connector_builder/impl/default_api.py

```diff
@@ -105,15 +109,22 @@ async def read_stream(self, stream_read_request_body: StreamReadRequestBody = Bo
         Using the provided manifest and config, invokes a sync for the specified stream and returns groups of Airbyte messages
         that are produced during the read operation
         :param stream_read_request_body: Input parameters to trigger the read operation for a stream
+        :param limit: The maximum number of records requested by the client
```

Comment:
should we explicitly define the range as [1, max_record_limit]?

Reply:
Yes, good call. Added.